packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/SynapseTableCell.tsx

```diff
@@ -192,7 +193,7 @@ function SynapseTableCell(props: SynapseTableCellProps) {
           {jsonData.map((val: string, index: number) => {
             return (
               <React.Fragment key={val}>
-                {val}
+                <Linkify text={val} />
```

Comment:
Failing test related to handling INTEGER_LIST here, you could probably either not `Linkify` INTEGER_LIST or change the test to be more permissive